datadog/resource_datadog_integration_aws.go

```diff
@@ -95,6 +95,14 @@ func resourceDatadogIntegrationAws() *schema.Resource {
 				"resource_collection_enabled": {
 					Type:         schema.TypeString,
 					Description:  "Whether Datadog collects a standard set of resources from your AWS account.",
+					Deprecated:   "Deprecated in favor of `extended_resource_collection_enabled`.",
+					Computed:     true,
+					Optional:     true,
+					ValidateFunc: validation.StringInSlice([]string{"true", "false"}, true),
+				},
+				"extended_resource_collection_enabled": {
+					Type:         schema.TypeString,
+					Description:  "Whether Datadog collects additional attributes and configuration information about the resources in your AWS account. If `cspm_resource_collection` is enabled, this will be automatically enabled as well.",
```

Comment:
The wording we used in the api-spec was "Required for `cspm_resource_collection`" which I think gets across the idea that we'll fail if it's set to false while cspm is set to true. I don't feel strongly about the wording, but it'd be nice for them to be consistent. Do you think this wording should be in the api-spec too?

Reply:
I can change the tf description to match api spec, I wanted to mention that the value might be enabled automatically depending on the cspm setting, but since the field is computed I think that might not be necessary to explicitly point out.